ggshield/cmd/iac/scan/diff.py

```diff
@@ -111,9 +112,21 @@ def iac_scan_diff(
 ) -> Optional[IaCDiffScanResult]:
     config = ctx.obj["config"]
     client = ctx.obj["client"]
-
+    verbose = config.user_config.verbose if config and config.user_config else False
+    if verbose:
+        click.echo(f"> Scanned files in reference {ref}")
```

Comment:
We should only display files that are IaC IMO.

Reply:
Isn't it supposed to show all files which were sent to GIM?